apps/vs-code-react/src/main.tsx

```diff
@@ -4,21 +4,40 @@ import { store } from './state/store';
 import { ThemeProvider } from './themeProvider';
 import { WebViewCommunication } from './webviewCommunication';
 import { initializeIcons } from '@fluentui/react';
+import { getReactQueryClient } from '@microsoft/logic-apps-designer';
 import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
+import { IntlProvider } from 'react-intl';
+import { QueryClientProvider } from 'react-query';
 import { Provider } from 'react-redux';
 
 initializeIcons();
+
+const queryClient = getReactQueryClient();
+
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 const container = document.getElementById('root')!;
 const root = createRoot(container);
 root.render(
   <StrictMode>
     <ThemeProvider>
       <Provider store={store}>
-        <WebViewCommunication>
-          <Router />
-        </WebViewCommunication>
+        <QueryClientProvider client={queryClient}>
```

Comment:
I got a feeling that if we have a general queryClient provider, each of the  apps inside the Router component shouldn't have another queryClient provider. Check for example this one `apps/vs-code-react/src/app/overview/app.tsx` This can be addressed in another PR